### ISSUE TITLE
Automatically download extension at the time of restoring the backup

### ISSUE THREAD

**Issue by AnujSsStw** [tachiyomiorg/tachiyomi#6338]
### Describe your suggested feature

So basically when you restore the backup It should have to option like do want to install all the previews extensions.

Overall it removes the hassle of again downloading all that thing at the time of switching devices.

### Other details

- add extension check box option in **Create backup** option.
- if extensions are already downloaded then do nothing if not then download them.

maybe like this!
![photo1639227783](https://user-images.githubusercontent.com/92370865/145677563-33020e69-de64-468f-a0f9-3476e5a8755d.jpeg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by manueldidonna** [comment]
I agree, I have to wipe my phone quite often because I like to try different roms and it's very annoying to manually download the extensions. It would be great if the app allowed to save also the settings option (for example custom columns number in the library or the appearance)

**Comment by rileyhawk1417** [comment]
It would be nice to have all configurations backed up at once. That way you can save the backup and restore it on any device.

**Comment by MattyCZE95** [comment]
I agree too. I found my self to install all my 20 extension for 5th time already this year..
